app/models/piece.rb

```diff
@@ -0,0 +1,5 @@
+class Piece < ActiveRecord::Base
+
+  belongs_to :game
+  belongs_to :user
```

Comment:
Following on from @bmw95 slack comments, instead of changing this is belongs_to :player, we will rename the column to user_id the piece model.